src/CloningHandler.java

```diff
@@ -0,0 +1,16 @@
+
+import java.io.*;
```

Comment:
doesn't use auto import
https://stackoverflow.com/questions/147454/why-is-using-a-wild-card-with-a-java-import-statement-bad